metron-deployment/roles/metron_streaming/tasks/main.yml

```diff
@@ -130,6 +130,8 @@
     - { regexp: "bolt.hdfs.rotation.policy=", line: "bolt.hdfs.rotation.policy={{ metron_hdfs_rotation_policy }}" }
     - { regexp: "bolt.hdfs.rotation.policy.count=", line: "bolt.hdfs.rotation.policy.count={{ metron_hdfs_rotation_policy_count}}" }
     - { regexp: "bolt.hdfs.rotation.policy.units=", line: "bolt.hdfs.rotation.policy.units={{ metron_hdfs_rotation_policy_units }}" }
+    - { regexp: "storm.topology.enrichment.workers=", 
+          line: "storm.topology.enrichment.workers={{ storm_topology_enrichment_workers if storm_topology_enrichment_workers is defined else 1 }}" }
```

Comment:
Rather that use ```is defined``` here, you can set the default in metron_streaming/defaults/main.yml by adding ```storm.topology.enrichment.workers: 1``` somewhere in the file.